hello-world-notification-microservice/README.md

```diff
@@ -72,4 +72,43 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+## Device creation and measurement sending
+
```

Comment:
Below, I'd remove 'out' form '... , deletion sending out measurements' so it's just '..., deletion and sending measurements'